src/structure_managers/adaptor_neighbour_list.hh

```diff
@@ -973,8 +988,8 @@ namespace rascal {
     // solve inverse problem for all multipliers
     auto cell_inv{cell.inverse().eval()};
     auto multiplicator{cell_inv * xpos.eval()};
-    auto xmin = multiplicator.rowwise().minCoeff();
-    auto xmax = multiplicator.rowwise().maxCoeff();
+    auto xmin = multiplicator.rowwise().minCoeff().eval();
```

Comment:
Damn, how did that ever work - or did that change with the version of Eigen?

Reply:
yes I ended up with a problem here and needed to eval() the expressions

Reply:
note that xmin is not a double.